login-workflow/example3/src/components/DebugComponent.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Button, FormControl, InputLabel, MenuItem, Select, SelectChangeEvent, Typography } from '@mui/material';
+import { Link } from 'react-router-dom';
+import React, { useState } from 'react';
+import { useApp } from '../contexts/AppContextProvider';
+import { routes } from '../navigation/Routing';
+
+const LinksWrapperStyles = {
+    textAlign: 'center',
+    pb: 4,
+};
+
+export const DebugComponent = (): JSX.Element => {
+    const { language, setLanguage } = useApp();
+    const [debugMode, setDebugMode] = useState(false);
+    const [selectedLanguage, setSelectedLanguage] = useState(language);
+
+    const changeAppLanguage = (event: SelectChangeEvent): void => {
+        const appLanguage = event.target.value;
+        setSelectedLanguage(appLanguage);
+        setLanguage(appLanguage);
+    };
+
+    return (
+        <Box>
+            <Box
+                sx={{
+                    display: 'flex',
+                    flexDirection: 'row',
+                    justifyContent: 'space-between',
+                    alignItems: 'center',
+                }}
+            >
+                <Box>{debugMode && <Typography variant={'h6'}>DEBUG MODE</Typography>}</Box>
+                <Button
```

Comment:
Make this button position: absolute so it doesn't cause the login screen to scroll when the debug controls are hidden